### DISCUSSION TITLE
help to run t-pot

### DISCUSSION THREAD

**Discussion by reemoove** [telekom-security/tpotce#1121] [Q&A] [answered]
**Hello guys I chose t-pot as a graduation project  I installed it on vmware but I don't know what should I do after installing ,I need to configure it as mysql server or anything else and I want to show my supervisor that t-pot is working well , may anyone help me in that ? and thanks a lot .**

**Comment by t3chn0m4g3** [comment, marked as answer]
I recommend to start with the [Readme](https://github.com/telekom-security/tpotce/blob/master/README.md) and follow up on the links of the individual honeypots and the technologies in [use](https://github.com/telekom-security/tpotce/discussions/995) (Docker, ELK stack, etc.). If you search Google / Youtube for `tpotce` you will also find lot of great guides and videos from many talented people explaining how they adopted T-Pot as their project or work environment. For some artificial data simply run `/opt/tpot/bin/hptest.sh 127.0.0.1`.
